src/flint/fmpz.jl

```diff
@@ -1339,7 +1381,12 @@ function moebius_mu(x::fmpz)
                     (Ref{fmpz},), x))
 end
 
-moebius_mu(x::Integer) = moebius_mu(fmpz(x))
+@doc Markdown.doc"""
+    moebius_mu(x::fmpz)
+> Returns the Moebius mu function of $x$ as an `Int`. The value
+> returned is either $-1$, $0$ or $1$. If $x < 0$ we throw a `DomainError()`.
+"""
+moebius_mu(x::Int) = moebius_mu(fmpz(x))
```

Comment:
convert to `Int`

Reply:
Hopefully it already returns an Int. But I'll check.

Reply:
It already returns an Int.

Reply:
Right, sorry.